src/modules/token/fungible/hooks/queries/useAppsMetaTokens.js

```diff
@@ -27,6 +27,7 @@ export const useAppsMetaTokensConfig = () => {
     method: 'get',
     event: 'get.blockchain.apps.meta.tokens',
     ...customConfig,
+    // TODO: Remove hardcoded network
     params: { limit, network: 'devnet' || network, chainID, ...(customConfig?.params || {}) },
```

Comment:
instead of having `'devent'` here we can just consume `DFAULT_NETWORK` constants which is in config.js